src/main/java/dev/symphony/harmony/mixin/redstone/ChangeRedstoneLampDelay.java

```diff
@@ -0,0 +1,43 @@
+package dev.symphony.harmony.mixin.redstone;
+
+import dev.symphony.harmony.config.HarmonyConfig;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.RedstoneLampBlock;
+import net.minecraft.state.property.BooleanProperty;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+// FEATURE: Removed Redstone Lamp delay
+// AUTHORS: Randomvideos
+@Mixin(RedstoneLampBlock.class)
+public abstract class ChangeRedstoneLampDelay extends Block {
+
+    @Final @Shadow
+    public static BooleanProperty LIT;
+
+    public ChangeRedstoneLampDelay(Settings settings) {super(settings);}
+
+    //Set the delay to redstoneBulbDelay(default 0) in scheduleBlockTick
+    @Inject(method = "neighborUpdate",at = @At("HEAD"), cancellable = true)
```

Comment:
Injecting at head and using `ci.cancel()` is generally unfavorable as it causes other modder's changes to completely fail to apply without any warning of a conflict.

There are several things you can do here, but my recommendation would be to use `@WrapOperation` at `"INVOKE"` targeting the call to `scheduleBlockTick()`. That call will then be passed into the arg `original`, which you can call by default if the config option is disabled.

If the config option is *enabled*, you can just call `scheduledTick()` directly. No need to copy and paste any code from inside that method, as `scheduledTick()` gets called as part of the schedule block tick process anyway.